views/applicationForms/applicationType.ejs

```diff
@@ -1,76 +1,181 @@
-<% pageTitle = "Choose a service" %>
+<% pageTitle="Choose a service" %>
 
-<div class="inner_header <%if (!user_data.loggedIn){%>no-user-signed-in<%}%>" >
+    <div class="inner_header no-user-signed-in">
+        <%if(back_link){%>
+            <a href="<%= back_link%>" class="back-to-previous govuk-link">Back</a>
+        <%}%>
+        <% if (user_data.loggedIn) { %>
+            <%- partial ('../partials/inner-header.ejs')%>
+        <% }else{%>
+            <div class="inner-header-content">
+                <span class="email-logout-text">
+                     <a  href="<%= userServiceURL%>/sign-in?next=serviceSelector&from=start" class="govuk-link" id="sign-in-link" >Sign in</a>
+                </span>
 
-    <%if(back_link){%> <a href="<%= back_link%>" class="back-to-previous govuk-link">Back</a><%}%>
-    <% if (user_data.loggedIn) { %>
-        <%- partial ('../partials/inner-header.ejs')%>
-    <% }else{%>
-        <div class="inner-header-content">
-            <span class="email-logout-text">
-                 <a  href="<%= userServiceURL%>/sign-in?next=serviceSelector&from=start" class="govuk-link" id="sign-in-link" >Sign in</a>
-            </span>
-
-        </div>
-    <%} %>
-</div>
-
-<h1 class="heading-xlarge">Choose a service</h1>
-
-<div class="column-two-thirds">
-    <%if(user_data.user && user_data.user.dropOffEnabled){%>
-        <div class="form-group">
-            <h2 class="heading-medium">Business drop-off service</h2>
-            <ul>
-                <li>£30 per document</li>
-                <li>Milton Keynes drop-off and collection only</li>
-                <li>overnight service</li>
-                <li>the counter opens on working days from 1pm to 2pm</li>
+            </div>
+        <%} %>
+    </div>
 
-            </ul>
-            <a class="govuk-button govuk-button--secondary" id="dropoff-service" href="/new-application?app_type_group=3">Start a business drop-off application</a>
+    <%if (errorMessageExists) { %>
```

Comment:
something to consider: split the error message block out into a fragment / partial so you can import it in any page where you need an error message to be displayed RATHER THAN using the HTML for the error block in each page.  You can then have a single line where you pass the error variable (error message) to the fragment and your page will contain just the page-specific information rather than boilerplate guff.

Reply:
This is a good idea. As this is a bit of a bigger change involving multiple pages on the site, (if it were to be done properly that is). Would it make sense to create a separate ticket for this to be done later?

Reply:
yeah sure.